ts/BUILD.typescript

```diff
@@ -17,19 +18,65 @@ link_npm_package(
     visibility = ["//visibility:public"],
 )
 
+npm_package(
```

Comment:
These are all worker dependencies? Should we put them down below under ` # tsc worker`?

Reply:
Yes. Done